AI_ML/NLP/COUNT_VECTORIZER/COUNT_VECTORIZER_test_.py

```diff
@@ -43,17 +42,17 @@ def test_COUNT_VECTORIZER():
     res = COUNT_VECTORIZER.COUNT_VECTORIZER([element_a], {})
 
     # check that the outputs look correct
-    assert (res.type) == "ordered_pair"
+    assert isinstance(res, OrderedPair)
```

Comment:
This test has to be updated to reflect the new multiple output format of the node